### PR TITLE
fix: update acp chromium

### PR DESCRIPTION
**Pull request by RafaelTaranto**

### CODE REVIEW

**Comment by RafaelTaranto** [comment]
[Improve Chromium screen maximisation on Aaeon ACP](https://trello.com/c/fArJ03pw/20-improve-chromium-screen-maximisation-on-aaeon-acp)

File: deploy/codebase/updateinit.js [hunk 1]
```diff
@@ -72,7 +94,7 @@ async.series([
   async.apply(command, `cp -PR /tmp/extract/package/subpackage/lamassu-machine ${applicationParentFolder}`),
   async.apply(command, `cp -PR /tmp/extract/package/subpackage/hardware/${hardwareCode}/node_modules ${applicationParentFolder}/lamassu-machine/`),
   async.apply(installDeviceConfig),
-  async.apply(command, 'systemctl enable supervisor'),
```
Comment (joshmh) [inline deploy/codebase/updateinit.js]: Why is this line deleted?

Comment (RafaelTaranto) [reply deploy/codebase/updateinit.js]: Its something from another PR, fixed on rebase.
